toolkit/tools/imagegen/attendedinstaller/views/installerview/installerview.go

```diff
@@ -63,15 +63,42 @@ func New(calamaresInstallFunc func()) *InstallerView {
 	if err != nil {
 		logger.Log.Debugf("Calamares not found, defaulting to terminal based installer")
 	} else {
-		logger.Log.Debugf("Calamares found, but the gui installer is temporarily disabled until stabilized")
-		// iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+		iv.installerOptions = append(iv.installerOptions, uitext.InstallerGraphicalOption)
+
+		err = AssignDbusPermissions()
+		if err != nil {
+			logger.Log.Debugf("An error occured during reassignment of dbus permissions")
+		}
 	}
 
 	iv.needsToPrompt = (len(iv.installerOptions) != 1)
 
 	return iv
 }
 
+// This function is a workaround to deal with squashed,required permissions
+// within the iso_initrd environment. kpmcore requires that the
+// /usr/libexec/dbus-daemon-lauch-helper file have the group messagebus with
+// permissions 4750 in order to run properly. kpmcore is used by calamares to
+// power the partition module used by the gui installer to configure disk
+// selection.
+// Information on issue here: https://invent.kde.org/system/kpmcore/-/issues/15
+func AssignDbusPermissions() (err error) {
+	logger.Log.Debugf("Running chgrp for dbus-daemon-launch-helper")
+	cmd := exec.Command("chgrp", "messagebus", "/usr/libexec/dbus-daemon-launch-helper")
+	err = cmd.Run()
+	if err != nil {
+		logger.Log.Debugf("Error while running chgrp for dbus-daemon-launch-helper")
+	}
```

Comment:
Let's also print the error:
```suggestion
	if err != nil {
		logger.Log.Debugf("Error while running chgrp for dbus-daemon-launch-helper: %s", err)
	}
```

Reply:
Added